tf/environments/dev/main.tf

```diff
@@ -617,6 +617,16 @@ module "ooni_fastpath" {
     protocol    = "tcp",
     cidr_blocks = module.network.vpc_subnet_private[*].cidr_block,
     }, {
+    from_port   = 8475, # for serving jsonl files
+    to_port     = 8475,
+    protocol    = "tcp",
+    cidr_blocks = module.network.vpc_subnet_private[*].cidr_block,
+    }, {
+    from_port   = 80,
+    to_port     = 80,
+    protocol    = "tcp",
+    cidr_blocks = ["0.0.0.0/0"], # for dehydrated 
```

Comment:
Do we need dehydrated running on this host since we should be using AWS for managing TLS?

Reply:
I would say we want to do it in here so that we can deploy everything together in a secure way.